src/classes/CW_BatchJobProgressPageControllerTest.cls

```diff
@@ -0,0 +1,56 @@
+/**
+* File Name   :    CW_BatchJobProgressPageControllerTest.cls
+* Description :    B-110984
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 10/04/2014 Klea Kolaric	Initial version
+*
+*
+*/
+@isTest
+private class CW_BatchJobProgressPageControllerTest {
+
+    static testMethod void basicTest()
```

Comment:
Style issue: test methods should be annotated with @IsTest for better readibility